cajita/src/Cajita_FastFourierTransform.hpp

```diff
@@ -28,286 +28,548 @@ namespace Cajita
 namespace Experimental
```

Comment:
I think we are nearing the point where these features could live outside the `Experimental` namespace. Should we take that step in this PR, or hold off?

Reply:
As we discussed, we'll hold off on this for now